sdk/python/jobs/automl-standalone-jobs/automl-nlp-text-classification-multiclass-task-sentiment-analysis/automl-nlp-text-classification-multiclass-task-sentiment-mlflow.ipynb

```diff
@@ -1,517 +1,900 @@
-    "my_validation_data_input = Input(type=AssetTypes.MLTABLE, path=validation_mltable_path)\n",
-    "\n",
-    "# WITH REMOTE PATH: If available already in the cloud/workspace-blob-store\n",
-    "# my_training_data_input = Input(type=AssetTypes.MLTABLE, path=\"azureml://datastores/workspaceblobstore/paths/my_training_mltable\")\n",
-    "# my_validation_data_input = Input(type=AssetTypes.MLTABLE, path=\"azureml://datastores/workspaceblobstore/paths/my_validation_mltable\")"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "# 3. Configure and run the AutoML NLP Text Classification Multiclass training job\n",
-    "In this section we will configure and run the AutoML job, for training the model.    "
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {
-    "tags": [
-     "parameters"
-    ]
-   },
-   "outputs": [],
-   "source": [
-    "# general job parameters\n",
-    "compute_name = \"gpu-cluster\"\n",
-    "exp_name = \"dpv2-nlp-text-classification-experiment\"\n",
-    "dataset_language_code = \"eng\""
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {
-    "gather": {
-     "logged": 1634852262026
-    },
-    "jupyter": {
-     "outputs_hidden": false,
-     "source_hidden": false
-    },
-    "name": "text-classification-configuration",
-    "nteract": {
-     "transient": {
-      "deleting": false
-     }
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "# AutoML - Sentiment Analysis scenario: Train \"the best\" NLP Text Classification Multi-class model for a 'Sentiment Labeled Sentences' dataset.\n",
+        "\n",
+        "**Requirements** - In order to benefit from this tutorial, you will need:\n",
+        "- A basic understanding of Machine Learning\n",
+        "- An Azure account with an active subscription. [Create an account for free](https://azure.microsoft.com/free/?WT.mc_id=A261C142F)\n",
+        "- An Azure ML workspace. [Check this notebook for creating a workspace](../../../resources/workspace/workspace.ipynb) \n",
+        "- A Compute Cluster. [Check this notebook to create a compute cluster](../../../resources/compute/compute.ipynb)\n",
+        "- A python environment\n",
+        "- Installed Azure Machine Learning Python SDK v2 - [install instructions](../../../README.md) - check the getting started section"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "# 1. Connect to Azure Machine Learning Workspace\n",
+        "\n",
+        "The [workspace](https://docs.microsoft.com/en-us/azure/machine-learning/concept-workspace) is the top-level resource for Azure Machine Learning, providing a centralized place to work with all the artifacts you create when you use Azure Machine Learning. In this section we will connect to the workspace in which the job will be run.\n",
```

Comment:
Let's not include locale information in the url. Please remove en-us

Reply:
en-us removed for all three notebooks in weblinks